repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/AssertFixerRepair.java

```diff
@@ -19,27 +19,30 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 
-public class AssertFixerRepair extends AbstractStep {
+public class AssertFixerRepair extends AbstractRepairStep {
+    protected static final String TOOL_NAME = "AssertFixer";
```

Comment:
Frankly I don't get the benefit of one construction compared to the other one. Actually I think I prefer mine (because it does not involve to change anything :p) and because I think it's more explicit to implement the proper method with the name.

Reply:
Moreover we can define an interface to represent what's an AbstractRepairStep.